handlers/generateNodeStubs.js

```diff
@@ -21,25 +21,32 @@ const temporary_paths = {
 export const handler = async (event) => {
   try {
     console.log(`Generate proto :: ${JSON.stringify(event)}`);
-    var output = getHostAndKeyFromUrl(event["output_s3_path"]);
+    if (event["output_s3_path"].length > 0) {
```

Comment:
don't hardcode values. 
create a file awsEvents.js under uitls. 
and create an enum for s3 events
```
export  const s3Events = {
"OUTPUT_S3_PATH":"output_s3_path"
}
```